site/src/pages/components/accordion.explainer.mdx

```diff
@@ -0,0 +1,157 @@
+---
+menu: Proposals
+name: Exclusive Accordion (Explainer)
+path: /components/accordion.explainer
+pathToResearch: /components/accordion.research
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- [@dbaron](https://github.com/dbaron)
+- Last updated: April 21, 2023
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+## Table of Contents
+
+- [Background](#background)
+- [Developer Demand for Exclusive Accordion](#developer-demand-for-exclusive-accordion)
+- [Proposal](#proposal)
+  - [Open design questions](#open-design-questions)
+    - [Name](#name)
+    - [Interaction with `open` attribute](#interaction-with-open-attribute)
+    - [Use case for non-exclusive accordion semantic?](#use-case-for-non-exclusive-accordion-semantic)
+  - [Possibly-resolved design questions](#possibly-resolved-design-questions)
+    - [Scope](#scope)
+- [Alternatives considered](#alternatives-considered)
+  - [CSS Toggles](#css-toggles)
+
+# Background
+
+A **disclosure** widget
+([research](/components/disclosure.research))
+is a UI that shows a summary or heading along
+with an indicator (sometimes a triangle or a plus sign) that allows the
+widget to be expanded to show additional details.
+An **accordion**
+([research](/components/accordion.research))
+is a sequence of related disclosure widgets.
+Some accordions are **exclusive accordions**,
+which means that at most one of the disclosures in that accordion
+can be open at the same time.
+
+The existing HTML
+[`details` element](https://html.spec.whatwg.org/multipage/C#the-details-element)
+represents a disclosure widget.
+A sequence of `details` elements can be used for a non-exclusive accordion.
+However, there is not a current web platform feature
+for exclusive accordions.
+
+It is worth noting that the HTML `details` element
+is not currently as styleable as it needs to be to meet many use cases.
+Improving styleability of `details` will be addressed in a separate proposal,
+intended to address both better ability to style/replace the disclosure triangle,
+and (maybe separately) to fit the parts of a `details`
+into different types of layout (such as grid).
+
+# Developer Demand for Exclusive Accordion
+
+Design systems that offer accordion widgets vary
+as to whether those widgets are exclusive or not.
+Based on the systems documented in the
+[accordion research](/components/accordion.research),
```

Comment:
The current accordion research doesn't show much...

Reply:
Yes, it doesn't show much... though it does have one sentence saying which of the systems listed are exclusive or not, and it has links to demo pages for each of them.

(I didn't take the time to figure out how to create images... and it's also not clear how useful images would be for this question since they don't show the interaction behavior.)